Makefile

```diff
@@ -1,19 +1,26 @@
 #!/usr/bin/make
 
+TOX_VER = $(shell tox --version 2>/dev/null || echo "0.0")
+TOX_VER_MAJOR := $(shell echo $(TOX_VER) | cut -f1 -d.)
+TOX_VER_MINOR := $(shell echo $(TOX_VER) | cut -f2 -d.)
+TOX_GT_1_8 := $(shell [ $(TOX_VER_MAJOR) -gt 1 -o \( $(TOX_VER_MAJOR) -eq 1 -a $(TOX_VER_MINOR) -ge 8 \) ] && echo true)
+
 all: lint unit_test
 
 
 .PHONY: clean
 clean:
 	@rm -rf .tox
 
-.PHONY: apt_prereqs
-apt_prereqs:
-	@# Need tox, but don't install the apt version unless we have to (don't want to conflict with pip)
-	@which tox >/dev/null || (sudo apt-get install -y python-pip && sudo pip install tox)
+.PHONY: tox_prereqs
+tox_prereqs:
+ifneq ($(TOX_GT_1_8),true)
+        @echo "Pip installing tox"
+        sudo apt-get install -y python-pip && sudo pip install -U 'tox>=1.8'
```

Comment:
I'm inclined to remove the tox dependency management from the layer entirely and fix the issue in charmbox, since that's our standard env for running tests and it is expected to manage most of the dependencies, of which the correct version of tox is one.

I'm going to close this PR and we'll open on against charmbox, and we can revisit removing the dep management from this layer entirely later.